public/assets/sass/layouts/_impact_discover.scss

```diff
@@ -228,13 +228,18 @@
 
         a {
             cursor: pointer;
+            display: flex;
+            align-items: center;
+            max-height: 100%;
+            width: 100%;
 
             img.bg-project {
-                position:absolute;
+                // position:absolute;
```

Comment:
I think it's better to remove lines than comment them. Otherwise, we end up with polluted code.

Reply:
True! Fixed in #328